imap_processing/codice/codice_l1b.py

```diff
@@ -47,6 +47,21 @@ def convert_to_rates(dataset: xr.Dataset, descriptor: str) -> np.ndarray:
         constants, f"{descriptor.upper().replace('-', '_')}_VARIABLE_NAMES"
     )
 
+    if descriptor.startswith("lo-"):
+        # Calculate energy_table using voltage_table and k_factor
+        energy_attrs = dataset["voltage_table"].attrs
+        energy_attrs["UNITS"] = "keV/e"
+        energy_attrs["LABLAXIS"] = "E/q"
+        energy_attrs["CATDESC"] = "Energy per charge"
+        energy_attrs["FEILDNAM"] = "Energy per charge"
```

Comment:
Can you create an "energy_table" attribute in the yaml file that inherits from voltage_table and replaces these values?

Otherwise, we should merge the dictionary attributes together rather than replace exisiting values.
```suggestion
        dataset["voltage_table"].attrs | {"UNITS": "keV/e", "LABLAXIS": "E/q", "CATDESC": "Energy per charge", "FEILDNAM": "Energy per charge"})
```

Reply:
I didn't want to call `cdf_attrs` just for this. That's why I replaced it. Cool. I like the merge idea.